app/controller/JobController.js

```diff
@@ -15,7 +15,20 @@ router.get('/job-roles', async (req, res) => {
         res.locals.errormassege = e
         res.render('jobRoleView')
     }
-    
+});
+
+//take request body and response body
+//await for data from api with sended band id, 
+//render bandCompView with data and band name if not error
+//if error occurres, render error message
+router.get('/band-comp/:bandName/:id', async (req, res) => {
```

Comment:
I don't think you'll need the band name here, just the band ID should be enough to get the information you need from the API

Reply:
Changed of url possible because with changes on backend we return band_name with competencies, fixed